lntest/node.go

```diff
@@ -882,16 +890,16 @@ func (hn *HarnessNode) lightningNetworkWatcher() {
 
 		req := &lnrpc.GraphTopologySubscription{}
 		ctx, cancelFunc := context.WithCancel(context.Background())
+		defer cancelFunc()
 		topologyClient, err := hn.SubscribeChannelGraph(ctx, req)
 		if err != nil {
-			// We panic here in case of an error as failure to
-			// create the topology client will cause all subsequent
-			// tests to fail.
-			panic(fmt.Errorf("unable to create topology "+
-				"client: %v", err))
+			msg := fmt.Sprintf("%s(%d): unable to create topology "+
```

Comment:
Won't the node name, ID and timestamp be in the log file anyway? Is this just left over from debugging or should this explicitly be shown in the itest log directly?

Reply:
In general they won't by default. This cashes out in actual tests to the return value of `net.NewNode()` and plenty of error checks for that only say "unable to create new node: %v". You can _usually_ figure out which node errored (mostly, it's the last one created) but having it explicitly named is better IMO.

OTOH, this was specifically written to detect one issue and arguably the correct place for prepending this would be in the NewNode call directly (so _any_ errors correctly identify the offending node not just this one). I can revert it if you feel strongly enough about it (or move to NewNode).